main.js

```diff
@@ -3,13 +3,45 @@ import { createHeaderComponent } from "./lb/titleComponent";
 import "./style.css";
 import createCharacterComponent from "./components/characterComponent";
 
-function createApp() {
+async function createApp() {
   const appElement = document.querySelector("#app");
 
   const headerElement = createHeaderComponent();
 
-  const mainElement = createElement("main", { className: "main" }, [
-    createCharacterComponent({
+  const response = await fetch("https://rickandmortyapi.com/api/character");
+  const body = await response.json();
+  const characters = body.results;
+
+  /*const characters = [
+    {
+      name: "Rick",
+      status: "Death",
+      race: "Alien",
+      location: "Earth",
+      image:
+        "https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse2.mm.bing.net%2Fth%3Fid%3DOIP.8OjjdUN8xCiccTkUtcwIHgHaJV%26pid%3DApi&f=1",
+    },
+    {
+      name: "Morty",
+      status: "Alive",
+      race: "Alien",
+      location: "Mars",
+      image:
+        "https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse2.mm.bing.net%2Fth%3Fid%3DOIP.mPRQa5i7FgrOmGOT2mvFNwHaHA%26pid%3DApi&f=1",
+    },
+  ];*/
+
+  const characterElement = characters.map((character) =>
```

Comment:
Maybe call it `characterElements` or `characterCards` since it is not just one character element, it's a whole array of characters :)
```suggestion
  const characterElements = characters.map((character) =>
```